build.gradle

```diff
@@ -409,8 +406,8 @@ apply plugin: 'nebula.ospackage'
 afterEvaluate {
     ospackage {
         packageName = "${name}"
-        release = isSnapshot ? "0.1" : '1'
-        version = "${project.version}" - "-SNAPSHOT"
```

Comment:
Is this removal of SNAPSHOT intended?

Reply:
You're faster than me, I didn't finish this pr :)

Reply:
haha I was just reviewing all of the changes. Should have done it in a single review.